pages/dashboard.js

```diff
@@ -280,10 +281,11 @@ const Dashboard = () => {
             borderRadius='0'
             color='ocean'
             fontSize='1.5rem'
-            onClick={() => downloadData(
-              JSON.stringify(userInstallData),
-              'flossbank_user_data.json'
-            )}
+            onClick={() =>
```

Comment:
idk how your commits pass but this breaks the pre commit hook of standard! not sure why it keeps getting changed :( i can't even commit when it's like this

Reply:
I'm not sure what's going on here TBH. :\

Reply:
what happens when you run `npx standard` with this code, does it yell at you that indentation is wrong?

Reply:
Had to temporarily disable format on save in VS code until I can find a better solution.

Reply:
> what happens when you run `npx standard` with this code, does it yell at you that indentation is wrong?

I already changed it so I can't run and check, but I haven't been getting warnings about indentation when committing.